tweet_bible_verse.py

```diff
@@ -40,8 +40,8 @@ def get_bible_verse():
     response.raise_for_status()
 
     soup = BeautifulSoup(response.content, "html.parser")
-    verse = soup.select(".votd-box > p")[0].text
-    verse_url = urljoin(BIBLE_VERSE_URL, soup.select(".votd-box > a")[0]['href'])
+    verse = soup.find_all(id="verse-text")[0].text
```

Comment:
You are spot on the css selector. However, if you use "soup.select_one" instead of "soup.find_all" then you can avoid the index 0 access. Could you please explore that and submit the changes.
```suggestion
    verse = soup.find_all(id="verse-text")[0].text
```